tools/hirondelle_crim_ogc_api_processes/crim_project_ogc_api_processes.xml

```diff
@@ -0,0 +1,221 @@
+				<param name="band_q" label="band_q" optional="false" value="" help="No description provided!" type="text"/>
+				<param name="band_r" label="band_r" optional="false" value="" help="No description provided!" type="text"/>
+				<param name="band_s" label="band_s" optional="false" value="" help="No description provided!" type="text"/>
+				<param name="band_t" label="band_t" optional="false" value="" help="No description provided!" type="text"/>
+				<param name="band_u" label="band_u" optional="false" value="" help="No description provided!" type="text"/>
+				<param name="band_v" label="band_v" optional="false" value="" help="No description provided!" type="text"/>
+				<param name="band_w" label="band_w" optional="false" value="" help="No description provided!" type="text"/>
+				<param name="band_x" label="band_x" optional="false" value="" help="No description provided!" type="text"/>
+				<param name="band_y" label="band_y" optional="false" value="" help="No description provided!" type="text"/>
+				<param name="band_z" label="band_z" optional="false" value="" help="No description provided!" type="text"/>
+			</when>
+			<when value="download-band-sentinel2-product-safe">
+				<param name="user_credentials" label="user_credentials" optional="false" type="data" format="json"/>
+				<param name="cookie" label="cookie" optional="false" value="" type="text"/>
+				<param name="product_url" label="product_url" optional="false" value="" help="No description provided!" type="text"/>
+				<param name="resolution_Array_text" label="resolution" optional="false" value="" help="Please provide comma-seperated values of type text here." type="text"/>
+				<param name="band" label="band" optional="false" value="" help="No description provided!" type="select">
+					<option value="B01">B01</option>
+					<option value="B02">B02</option>
+					<option value="B03">B03</option>
+					<option value="B04">B04</option>
+					<option value="B05">B05</option>
+					<option value="B06">B06</option>
+					<option value="B07">B07</option>
+					<option value="B08">B08</option>
+					<option value="B8A">B8A</option>
+					<option value="B09">B09</option>
+					<option value="B11">B11</option>
+					<option value="B12">B12</option>
+					<option value="AOT">AOT</option>
+					<option value="SCL">SCL</option>
+					<option value="TCI">TCI</option>
+					<option value="WVP">WVP</option>
+				</param>
+				<param name="debug_Array_boolean" label="debug" value="" help="Please provide comma-seperated values of type boolean here." type="text"/>
+			</when>
+			<when value="download-band-sentinel2-stac-item">
+				<param name="cookie" label="cookie" optional="false" value="" type="text"/>
+				<param name="product_url" label="product_url" optional="false" value="" help="No description provided!" type="text"/>
+				<param name="band" label="band" optional="false" value="" help="No description provided!" type="select">
```

Comment:
No description provided?

Reply:
Ah, this is confusing. I will remove it.